### ISSUE TITLE
How to resolve a gl_sc_reset() crash with libmpv

### ISSUE THREAD

**Issue by skrew** [mpv-player/mpv#9239]
Sometimes, specially after a glView resize / device rotation, when the video is paused and the user unpause it, the player crash in `shader_cache.c`:

```
// gl_sc_reset() must be called after ending the previous render process,
// and before starting a new one.
assert(!sc->needs_reset);
```

I never "end" a render process, as the video player is always displayed, the most "ending" process in my app is a mpv stop. 

- What is a "ending render process" ?
- Is there a way to call `gl_sc_reset()` via libmpv ?

Informations:
- libmpv 0.33.1
- iOS / tvOS
- Rendering video output: `libmpv`

**Comment by Dudemanguy** [comment]
I don't know what's going on here but I'm pretty sure this shouldn't be happening. If this is still a problem with 0.36, please open a new issue.
